openapi3.yaml

```diff
@@ -1180,13 +1500,16 @@ components:
         for filtering in API requests. Can be any descriptive name up to 50 characters.
     returnStage:
       type: boolean
+      example: false
```

Comment:
not sure we need exampels for stuff like boolean

Reply:
ok, removed